tests/test_sensor.cpp

```diff
@@ -68,7 +68,7 @@ THREAD_FUNCTION_RETURN_TYPE logTask(void*)
 
 
 
-int main(int, char*)
+int main()
```

Comment:
This comment is for the sake of argumentation.
I do not think it matters, but why not just putting `char**`?
IMHO this way you show that this is a traditional main function...
Maybe this is useless to have arguments that are not used.
I do not know what the C++ standard is saying about this.

Reply:
As far as I know the standard wants either `int main()` or `int main(int, char**)`.  If the arguments are unused, I typically use the empty version as it is more simplistic.

Reply:
Thanks for the information, you can merge, thanks for the warning hunt!!

Reply:
Actually for `clang` this is an error (which is why I found it).

Reply:
I am doing a bunch of warning hunt from time to time as well, but I try to make the basic (core-robotics) package clean right now.